classes/user/Repository.php

```diff
@@ -183,8 +183,10 @@ public function canUserAccessStage($stageId, $workflowType, $userAccessibleStage
     {
         $workflowRoles = Application::get()->getWorkflowTypeRoles()[$workflowType];
 
-        if (array_key_exists($stageId, $userAccessibleStages)
-            && !empty(array_intersect($workflowRoles, $userAccessibleStages[$stageId]))) {
+        if (
+            array_key_exists($stageId, $userAccessibleStages)
+            && !empty(array_intersect($workflowRoles, $userAccessibleStages[$stageId]))
+        ) {
```

Comment:
If this wasn't an automated fix, perhaps it's worth to simplify the code.

```suggestion
        if (!empty(array_intersect($workflowRoles, $userAccessibleStages[$stageId] ?? []))) {
```